content/millennium/r4/security/provenance.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Provenance | R4 API
+---
+
+# Provenance
+
+* TOC
+{:toc}
+
+## Overview
+
+The Provenance resource tracks the source of external data and is used to assess the trustworthiness and reliability of a resource. Sources can be an organization author or organization transmitter.  Entities are Continuity of Care DocumentReferences (CCDs).
+
+The following fields are returned if valued:
+
+* [Provenance id](http://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Target](http://hl7.org/fhir/R4/provenance-definitions.html#Provenance.target){:target="_blank"}
+* [Recorded](http://hl7.org/fhir/R4/provenance-definitions.html#Provenance.recorded){:target="_blank"}
+* [Agent type (author or transmitter)](http://hl7.org/fhir/R4/provenance-definitions.html#Provenance.agent.type){:target="_blank"}
+* [Agent role (source or informant)](http://hl7.org/fhir/R4/provenance-definitions.html#Provenance.agent.role){:target="_blank"}
+* [Agent who (Practitioner, Device, or Organization)](http://hl7.org/fhir/R4/provenance-definitions.html#Provenance.agent.who){:target="_blank"}
+* [Agent on behalf of](http://hl7.org/fhir/R4/provenance-definitions.html#Provenance.agent.onBehalfOf){:target="_blank"}
+* [Entity role](http://hl7.org/fhir/R4/provenance-definitions.html#Provenance.entity.role){:target="_blank"}
+* [Entity what (Continuity of Care DocumentReference id)](http://hl7.org/fhir/R4/provenance-definitions.html#Provenance.entity.what){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:provenance, :r4) %>
+
+## Search
+
+Search for Provenances that meet supplied query parameters:
+
+    GET /Provenance?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name         | Required? | Type          | Description
+--------------|-----------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------
+ `_id`        | This, or any other required search parameter | [`token`]     | The logical resource id associated with the resource. Example: `doc-7845`
+ `target`     | This and/or any other search param, or `_id` | [`reference`] | Target Reference(s) (usually version specific). Example: `DocumentReference/66547`
+
+### Headers
+
+<%= headers fhir_json: true %>
+
+### Example
+
+#### Request
+
+    GET https://fhir-open.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Provenance?_id=doc-881057
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_provenance_bundle) %>
+
+<%= disclaimer %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## Retrieve by id
+
+List an individual Provenance by its id:
+
+   GET /Provenance/:id
```

Comment:
nit: one more space of indentation

Reply:
https://github.com/cerner/fhir.cerner.com/pull/371/commits/5ce832aef31ae253bd4fd5c121b1cfc741a1491b#diff-ced18538a7ebc9d99f07afaacf776c83R72